src/status_im2/common/scroll_page/view.cljs

```diff
@@ -41,65 +28,98 @@
    (max minimum)
    (min maximum)))
 
-(defn icon-top-fn
-  [scroll-height]
-  (if (<= scroll-height negative-scroll-position-0)
-    -40
-    (->> (+ scroll-position-0 scroll-height)
-         (* (if platform/ios? 3 1))
-         (+ -40)
-         (min 8))))
-
-(defn icon-size-fn
-  [scroll-height]
-  (->> (+ scroll-position-0 scroll-height)
-       (* (if platform/ios? 3 1))
-       (- max-image-size)
-       (max min-image-size)
-       (min max-image-size)))
+(defn scroll-page-header
+  [scroll-height name page-nav cover sticky-header]
+  [:f>
+   (fn []
+     (let [input-range         (if platform/ios? [-47 10] [0 150])
+           output-range        (if platform/ios? [-100 0] [-169 -45])
+           y                   (animated/value @scroll-height)
+           translate-animation (animated/interpolate y
+                                                     {:inputRange  input-range
+                                                      :outputRange output-range
+                                                      :extrapolate (:clamp animated/extrapolate)})
+           opacity-animation   (reanimated/use-shared-value 0)
+           threshold           (if platform/ios? 30 170)]
+       (quo.react/effect!
```

Comment:
I tried using this one but it does not seem to work

Reply:
Could you share the details with us? If it doesn't work we should probably investigate why on a separate issue. It would be great if you could create a GH issue describing how to reproduce the problem.